content/pt-br/docs/reference/glossary/rbac.md

```diff
@@ -0,0 +1,18 @@
+---
+title: RBAC (Controle de Acesso Baseado em Funções)
+id: rbac
+date: 2018-04-12
+full_link: /docs/reference/access-authn-authz/rbac/
+short_description: >
+  Gerencia decisões de autorização, permitindo que os administradores configurem dinamicamente políticas de acesso por meio da API do Kubernetes.
+
+aka: 
```

Comment:
```suggestion
aka:
 - Role Based Access Control
 - Controle de Acesso Baseado em Funções
```
This is where I'd put the alternative name (and in the title, I'd leave it just as RBAC). What do you think?

Reply:
Nice. Thank you.